manim/mobject/text/code_mobject.py

```diff
@@ -25,8 +25,6 @@
 from manim.mobject.types.vectorized_mobject import VGroup
 from manim.utils.color import WHITE
 
-__all__ = ["Code"]
-
```

Comment:
Is there a reason why this was deleted? As far as I know, this should be kept.

Reply:
I noticed it was repeated from earlier (see lines 5-7). If there's a reason to have it twice (say, if the imports do something weird) then keep it, but afaik, it's redundant.